plotly/plotly_offline_aux/plotlyoffline.m

```diff
@@ -61,7 +61,7 @@
                       id, link_text);
     
     plotly_script = sprintf(['\n<div class="%s loading" style=', ...
-                              'color: rgb(50,50,50);">Drawing...</div>' ... 
+                              'color: rgb(50,50,50);"> </div>' ... 
```

Comment:
why not remove this entire empty DIV? seems like it's useless?

Reply:
Ok. I will review it

Reply:
@jackparmer please review again and let me know